R/bundle.R

```diff
@@ -45,6 +56,12 @@ maxDirectoryList <- function(dir, parent, totalSize) {
     contents <- contents[!grepl(glob2rx(".Rhistory"), contents)]
   }
 
+
+  # add packrat lock file to contents if it exists
+  # TODO: Handle the case where dir is a subdirectory of the project
+  if (file.exists(snapshotLockFile(dir)))
```

Comment:
Put this in the `nchar(parent) == 0` clause above so that it only runs at the root; in the case of deploying a directory which contains a packrat project in a subdirectory, we already include the lockfile (as the `packrat` folder is only excluded at the root)